binding/web/src/porcupine_worker_handler.ts

```diff
@@ -0,0 +1,95 @@
+/*
+  Copyright 2022 Picovoice Inc.
+
+  You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+  file accompanying this source.
+
+  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+  specific language governing permissions and limitations under the License.
+*/
+
+/// <reference no-default-lib="false"/>
+/// <reference lib="webworker" />
+
+import { Porcupine } from './porcupine';
+import { PorcupineWorkerRequest } from './types';
```

Comment:
in the binding I noticed we do either `{ Porcupine }` or `{Porcupine}`. To be consistent across other web bindings lets do `{ Porcupine }`

Reply:
Personally I like `{ Porcupine }` better but WordStorm apparantly prefers the other one; I agree that we should be concistent and reformatted all codes again to be sure that all imports are the same